benchmark/connections.json

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
We use [TOML](https://github.com/toml-lang/toml) instead of JSON for our config files. They are more readable and allow for comments.

Reply:
but this is more like input file, not config file, and it is easy to generate this file from some cli like tool

Reply:
It is even easier to parse toml in your app. Here is an example: https://github.com/spf13/viper#reading-config-files

TOML parsers are much less complex and error prone. TOML config file i more human readable and you can put comments in it.

Mainflux uses TOML as a convention. Please follow this convention.